src/modules/dirwatcher.js

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+import EventEmitter from 'events';
+import { differenceWith, isEqual } from 'lodash';
+
+export class DirWatcher extends EventEmitter {
+  constructor(props) {
+    super(props);
+    this.files = [];
+  }
+
+  watch(path, timeout) {
+    const watcherFunc = () => {
```

Comment:
Export this function out from watch method, please

Why? Because it will better for further testing.